sdks/python/apache_beam/io/gcp/datastore/v1new/datastoreio_test.py

```diff
@@ -177,6 +180,50 @@ def test_write_mutations_non_retryable_error(self):
           mock_throttler, rpc_stats_callback, throttle_delay=0)
     rpc_stats_callback.assert_called_once_with(errors=1)
 
+  def test_write_mutations_metric_on_failure(self):
```

Comment:
Please write a similar test for the read side:
invoking  _QueryFn

Reply:
Done!